types/html-to-text/index.d.ts

```diff
@@ -300,11 +300,17 @@ export interface FormatOptions {
      * (Only for: `anchor` formatter.) Ignore all links. Only process internal text of anchor tags.
      */
     ignoreHref?: boolean | undefined;
+    /**
+     * (Only for: `anchor` and `image` formatters.) Surround links with these brackets.
+     * Set to false or ['', ''] to disable.
+     */
+    linkBrackets?: [string, string] | boolean | undefined;
```

Comment:
Why `boolean` when the doc says `false`?  This appears to work and does not allow `true`:
```
linkBrackets?: [string, string] | false | undefined;
```

Reply:
Right. my bad. Changed it.